### ISSUE TITLE
ScaleFactor/DPI

### ISSUE THREAD

**Issue by bforlgreen** [ScottPlot/ScottPlot#3970]
There seems to be an issue with mouse interaction handling when setting the scale factor to be something higher than 1.

Repro steps :

- Leave Scale Factor to 1
- Middle mouse pan inside the axis to do a "locked axis pan"
- All good

- Now change Scale Factor to something like 1.5
- Middle mouse pan inside the axis to do a "locked axis pan"
- Notice it only works in certain areas, but for the most part, very broken

I was testing this with ScotPlot 5 (5.0.35) in WPF

```
private void WpfPlot1_Loaded(object sender, RoutedEventArgs e)
 {
     double[] dataX = { 1, 2, 3, 4, 5 };
     double[] dataY = { 1, 4, 9, 16, 25 };
     WpfPlot1.Plot.Add.Scatter(dataX, dataY);
     //WpfPlot1.Plot.ScaleFactor = 1.5;

     WpfPlot1.Refresh();
 }
```

**Comment by aespitia** [comment]
I think Scale Factor and DPI are separate things, from what I've seen there isn't a way to set a DPI in SkiaSharp, so if you need a specific DPI, i think you have to upscale it yourself.  I ran into this issue when saving images, I think the default DPI is something like 75dpi, which is not great for printing output.  you need something like 300dpi.

**Comment by MCF** [comment]
This comes up over and over.  For WPF here is some information about how to deal with it:

https://github.com/ScottPlot/ScottPlot/issues/3465#issuecomment-2010669913

There are other links to documentation and similar discussion in that issue, feel free to read through them to gain more background.

It isn't actually that complicated to deal with both getting the coordinates and scaling plots properly on high DPI displays, but we haven't done a great job of explaining it.  For what it's worth the closed (and unmerged) documentation issue linked in the issue above is the best way to deal with high DPI displays for WPF applications IMHO.  I have been using those two techniques on a production application for many months and it works perfectly with any windows scaling. 

For reference here is a link to the unmerged documentation PR (again, for WPF, it is 100% the correct thing to do for any display - high DPI/scaled or otherwise):

https://github.com/ScottPlot/ScottPlot.NET/pull/34

**Comment by bforlgreen** [comment]
@MCF the example I showed above demonstrates the bug. Its an issue inside the library, not a user configuration issue.

**Comment by MCF** [comment]
Apologies @bforlgreen, I've answered the question of mouse coordinate conversion so many times it has become too automatic.

I'm not able to reproduce your problem.  Although I'm not sure I understand what middle mouse pan as a locked axis pan means.  Do you mean drawing a zoom region/rectangle with the middle mouse button?  Or using the shift/control keys + middle mouse button to draw a vertical or horizontal region respectively?  FWIW, I'm finding those functions working the same with or without a scale factor applied.

**Comment by MCF** [comment]
Or is it middle click drag on top of the axis label area (outside data area)?  I can confirm that is not working the same way once ScaleFactor != 1.

Ironically I noted the same problem a few weeks ago, but wasn't aware of the feature of middle click and drag on the axis labels to do an axis locked zoom - possibly because I had never seen it work properly before as I always have ScaleFactor != 1 in my dev environment.

Here is my report FWIW:

https://github.com/ScottPlot/ScottPlot/issues/3897

**Comment by bforlgreen** [comment]
> Or is it middle click drag on top of the axis label area (outside data area)? I can confirm that is not working the same way once ScaleFactor != 1.

Yep, that is it.

**Comment by BrianAtZetica** [comment]
> I can confirm that is not working the same way once ScaleFactor != 1.

I think this was fixed by PR #3887

**Comment by bforlgreen** [comment]
> > I can confirm that is not working the same way once ScaleFactor != 1.
> 
> I think this was fixed by PR #3887

I can repro it in the current source

**Comment by bforlgreen** [comment]
I've started working on a pullrequest to fix this, it was actually really easy to fix and I have also cleared up the ScaleFactor usage
